modulefiles/gsi_noaacloud.intel.lua

```diff
@@ -0,0 +1,28 @@
+help([[
+]])
+
+prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
```

Comment:
Why is this line included?  Isn't the `gsi-addon-env` line sufficient?  The hera, hercules, jet, orion, and s4 intel modulefiles only include the `gsi-addon-env` line.